contracts/multisig-prover/src/error.rs

```diff
@@ -74,4 +74,7 @@ pub enum ContractError {
 
     #[error("payload does not match the stored value")]
     PayloadMismatch,
+
+    #[error(transparent)]
+    StellarXdrError(#[from] stellar_xdr::curr::Error),
```

Comment:
I don't see that used anywhere and don't think there should be a Stellar-specific error at this level